app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -397,6 +410,7 @@ def handle_unsupported_secondary_id(enrollment, response)
     )
     # send SMS and email
     send_enrollment_status_sms_notification(enrollment: enrollment)
+    # MW: I _think_ this is desired behavior?
     send_failed_email(enrollment.user, enrollment)
```

Comment:
The idea is that if you get a failure proofing, we'll just send you that, and the fact that we _also_ had you flagged as fraudulent is kind of irrelevant.